packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -1186,17 +1211,31 @@ class _SliverAppBarState extends State<SliverAppBar> with TickerProviderStateMix
     }
   }
 
+  void _updateStretchConfiguration() {
```

Comment:
My thinking was that I wanted to follow the pattern the same pattern for snap, especially since the land of app bars and persistent headers is already pretty complex. Do you think it is harder to follow this way?

Reply:
Ah if this is a pattern that's used elsewhere then I think it's best to stay consistent.